src/locales/cy/translation.json

```diff
@@ -314,6 +314,22 @@
         }
       }
     },
+    "pyiTriageSelectDevice": {
+      "title": "Ydych chi ar gyfrifiadur neu dabled ar hyn o bryd?",
```

Comment:
Where are these translations from? the exact content is still TBC so we probably want to just copy over the English here for now (so it doesn't look like these are done and finalised already)

Reply:
There's a follow up ticket for final content and Welsh - [PYIC-5254](https://govukverify.atlassian.net/browse/PYIC-5254)

Reply:
These came from DCMAW directly, thankfully it was easy to spot which page this relates to

Reply:
Usually when we don't have the final content/Welsh translations confirmed we just set the English content here for the time being (which is fine since this page won't be going live yet) - @MikeCollingwood could you do that for now? otherwise when we come back to this it may look like we've already done the Welsh translations

Reply:
These were the welsh translations from DCMAW but I can modify to english even so?

Reply:
Yeah I think best for now since this content is TBC